jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/JibPluginConfiguration.java

```diff
@@ -135,6 +150,15 @@ public PermissionConfiguration() {}
     @Nullable @Parameter private String credHelper;
 
     @Parameter private FromAuthConfiguration auth = new FromAuthConfiguration();
+
+    @Parameter private List<PlatformsConfiguration> platforms;
+
+    /** Configuration for {@code platforms} parameter. */
+    public FromConfiguration() {
+      platforms = new ArrayList<>();
```

Comment:
Can the `platforms` list not be empty?

If we were to initialize this to an empty list at line 154, would it break the default beahavior?

Reply:
The idea is to have a default platform amd64/linux in the platforms any time a user fails to specify the platforms.

Reply:
Another way to create a default singleton (immutable) list:
```java
PlatformsConfiguration platform = new PlatformsConfiguration();
platform.os = ...
platform.version = ...
platforms = Collections.singletonList(platform);
```

Reply:
Done